src/ui/public/agg_types/controls/interval.html

```diff
@@ -35,6 +35,6 @@
     class="form-control"
     name="interval"
     min="0"
-    input-whole-number
```

Comment:
I think this might be the only reference to input-whole-number - can you delete the file and corresponding test file?